src/Compilers/CSharp/Test/Symbol/Compilation/GetSemanticInfoTests.cs

```diff
@@ -2273,8 +2273,8 @@ public static void Main(string[] args)
                 }
                 else
                 {
-                    thread1.Start();
                     thread2.Start();
+                    thread1.Start();
```

Comment:
The OS makes no guarantee that the threads run in the order started, so why pretend?